web-client/src/app/state/setup/setup.query.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@angular/core';
+import { Query } from '@datorama/akita';
+import { environment } from 'src/environments/environment';
+import { SetupState, SetupStore } from './setup.store';
+
+@Injectable({ providedIn: 'root' })
+export class SetupQuery extends Query<SetupState> {
+  tokenIssuer = this.getValue().tokenIssuer.trim() || environment.tokenIssuer;
+  xrpIssuer = this.getValue().xrpIssuer.trim() || environment.xrpIssuer;
+  ledger = this.getValue().ledger.trim()
+    ? { ...environment.xrplClient, server: this.getValue().ledger }
+    : environment.xrplClient;
+
```

Comment:
Will we add other fields as needed like `name`, `tokenSymbol`? Right now, these 3 values are the only ones that needs to be "configurable"?

Reply:
Correct.
The name is already in the environment so shouldn't be configurable.
The tokenSymbol will probably be used yes